ViewModel/AuctionItemCollectionRenderer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SomethingDigital\AuctionItem\ViewModel;
+
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use SomethingDigital\AuctionItem\Model\AuctionItemRepository;
+
+/**
+ * View Model to display Auction Items
+ *
+ * Class AuctionItemCollectionRenderer
+ * @package SomethingDigital\AuctionItem\ViewModel
+ */
+class AuctionItemCollectionRenderer implements ArgumentInterface
```

Comment:
nit: The name `AuctionItemCollectionRenderer` isn't exactly accurate because the view model doesn't do any rendering. It just fetches the raw data and returns it so the name could be confusing. You could do something like `AuctionItemData` or just `AuctionItem`.